.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: Continuous integration
 
 on:
   push:
+    branches:
```

Comment:
Do we want to introduce commits from master to the integration branch like this?  I would think we add these commits in separate PRs, or if we can avoid it not at all.

Reply:
Sorry, oversight. I should have rebased the integration branch and then this first